backend/app/controllers/api/v1/games_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Api::V1::GamesController < ApplicationController
+  def index
+    if params[:new] == "true"
+      render :json => Game.with_new_state
+    end
+  end
+  def show
+    render :json => Game.find_by_id(params[:id])
```

Comment:
Why not `Game.find(params[:id])`? What happens if there is no game with that id?